### ISSUE TITLE
Climber profile bug - Negative numbers

### ISSUE THREAD

**Issue by explosivecuriosity** [theCrag/website#4172]
Hi there,

My crag climber profile seems to be counting backwards.
My photo count is at -1 and I'm being told I need to log -4 more ascents to unlock my grade pyramid.

<img width="279" alt="Screen Shot 2023-02-05 at 11 37 37 am" src="https://user-images.githubusercontent.com/124475523/216796415-89e63816-24bf-4b60-ae1a-190e3198fef5.png">
<img width="190" alt="Screen Shot 2023-02-05 at 11 37 46 am" src="https://user-images.githubusercontent.com/124475523/216796416-ddc69fae-d30b-42e7-a008-57e85fc50aa9.png">

My profile;
https://www.thecrag.com/climber/mattcastle

Thanks in advance for help with this issue.
Matt

**Comment by rouletout** [comment]
Sometimes the stats lag behind which may lead to funny numbers - fixed itself, clsoing

**Comment by explosivecuriosity** [comment]
Thanks.
The photo count has fixed itself.
The number of ascents required to unlock the grade pyramid is still
counting backwards though. I logged 3 more ascents today and it's now at -7
more ascents needed to unlock.

Thanks,
Matt

**Comment by scd** [comment]
Are you deleting ascents

**Comment by explosivecuriosity** [comment]
No, I've never deleted an ascent since creating my account.

**Comment by explosivecuriosity** [comment]
Hi Simon,

Just wondering if you've had a chance to look into this issue to resolve it?

There is still an issue with the counting on my profile.
I added a new profile picture today and deleted the existing picture. My
photo count is now at 2, even though I deleted the other picture and only
have 1 photo in my gallery.

Yesterday I logged 3 more ascents and my grade pyramid now says I need -10
more ascents to unlock the feature. Still counting in reverse.

As mentioned in my previous email, I've never deleted an ascent I've
created.

Thanks for your help.
Matt
